ios/brave-ios/Sources/Brave/Frontend/Browser/SearchResultAdClickedInfoBar.swift

```diff
@@ -0,0 +1,171 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import DesignSystem
+import Foundation
+import Shared
+import SnapKit
+import SwiftUI
+import UIKit
+
+struct SearchResultAdClickedInfoBarUX {
+  static let toastHeight: CGFloat = 100.0
+  static let toastPadding: CGFloat = 10.0
+  static let toastCloseButtonWidth: CGFloat = 20.0
+  static let toastLabelFont = UIFont.systemFont(ofSize: 15, weight: .semibold)
```

Comment:
You should probably not use a fixed font size, replace with `UIFont.preferredFont(forTextStyle: .subheadline)` and test with multiple font sizes

Reply:
Fixed